src/pybind/mgr/dashboard/base.html

```diff
@@ -17,9 +17,9 @@
     <link rel="stylesheet"
           href="{{ url_prefix }}/static/AdminLTE-2.3.7/dist/css/AdminLTE.min.css">
     <link rel="stylesheet"
-          href="{{ url_prefix }}/static/AdminLTE-2.3.7/dist/css/skins/skin-blue.min.css">
+          href="/static/AdminLTE-2.3.7/dist/css/skins/skin-blue.min.css">
     <link rel="stylesheet"
-          href="{{ url_prefix }}/static/AdminLTE-2.3.7/plugins/datatables/jquery.dataTables.css">
+          href="/static/AdminLTE-2.3.7/plugins/datatables/jquery.dataTables.css">
```

Comment:
this has removed the {{url_prefix}}, it should remain (and also appear in the datatables url below)